### ISSUE TITLE
Specify children property for React 18

### ISSUE THREAD

**Issue by fresonn** [sanishkr/react-progressive-graceful-image#6]
Using this:
```js
<ProgressiveImage src={src}>
    {(src: string, loading: boolean) => {
      return loading ? <Plug /> : <img src={src} alt={altText} className={classes.productImage} />
    }}
</ProgressiveImage>
```
I get this error:
```
Property 'children' does not exist on type 
'IntrinsicAttributes & IntrinsicClassAttributes<ProgressiveImage> & Readonly<ProgressiveImageProps>'.
```
Referring to React 18 docs:
If you have a component with children, you have to explicitly declare them.

But `ProgressiveImageProps` interface don't specify it

**Comment by sanishkr** [comment]
@Vlad-Shevliakov thanks for pointing it out. It will be helpful if you can provide a code sandbox to reproduce the issue. I'll take a look to fix the issue, meanwhile.

**Comment by sanishkr** [comment]
The above PR should fix it.
Released in `v0.7.0`. Please check [release notes](https://github.com/sanishkr/react-progressive-graceful-image/releases/tag/v0.7.0).
